doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,35 @@
 
 <h3>New features since last release</h3>
 
+* `qml.defer_measurements` can now be used with program capture enabled.
+  [(#6838)](https://github.com/PennyLaneAI/pennylane/pull/6838)
+
+  Using `qml.defer_measurements` with program capture enables many new features, including:
+  * Significantly richer variety of classical processing on mid-circuit measurement values.
+  * Using mid-circuit measurement values as gate parameters.
+
+  Functions such as the following can now be captured:
+
+  ```python
+  import jax.numpy as jnp
+
+  qml.capture.enable()
+
+  def processing_fn(m1, m2):
```

Comment:
Any particular reason this is extracted into a helper function?

Reply:
I was using a significantly more complicated function locally so it looked nicer, I'll remove it :)